examples/kubernetes/dynamic-provisioning/filesystem/storageclass.yaml

```diff
@@ -0,0 +1,28 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: fsx-sc
+provisioner: fsx.openzfs.csi.aws.com
+parameters:
+  volumeType: "filesystem" #REQUIRED
+  kmsKey: "12345678-90ab-cdef-ghij-klmnopqrstuv"
+  automaticBackupRetentionDays: "1"
+  copyTagsToBackups: "false"
+  copyTagsToVolumes: "false"
+  dailyAutomaticBackupStartTime: "00:00"
+  deploymentType: "SINGLE_AZ_1" #REQUIRED
+  diskIopsConfiguration: "Iops=300,Mode=USER_PROVISIONED"
+  rootVolumeConfiguration: "CopyTagsToSnapshots=true,DataCompressionType=NONE,NfsExports=[{ClientConfigurations=[{Clients=*,Options=[rw,crossmnt]}]}],ReadOnly=false,RecordSizeKiB=128,UserAndGroupQuotas=[{Id=1,StorageCapacityQuotaGiB=10,Type=USER}]"
+  throughputCapacity: "64" #REQUIRED
+  weeklyMaintenanceStartTime: "7:09:00"
+  securityGroupIds: "sg-068000ccf82dfba88"
+  subnetIds: "subnet-0eabfaa81fb22bcaf" #REQUIRED
+  tags: "Tag1=Value1,Tag2=Value2"
+  skipFinalBackup: "true" #REQUIRED
```

Comment:
Based on our earlier conversation, ```skipFinalBackup``` should probably be ```deleteSkipFinalBackup```. Or maybe ```skipFinalBackupOnDeletion```. I think I prefer the latter, but let me know what you think.

Reply:
Should we comeback and fix the example when we implement the change? I feel like we should leave it as is for now since the example files should be a known working example.

Reply:
Oh, yeah that's a good point. Probably best to change this later.